🐙 source-monday: run up-to-date pipeline [2025-05-31]

airbyteio opened the pull request:
# Update source-monday

This PR was autogenerated by running `airbyte-ci connectors --name=source-monday up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- Update versions of libraries in poetry.: Failed

- PATCH bump source-monday version to 2.3.1: Successful

- Build source-monday docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Successful

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful




## Dependency updates

We use [`syft`](https://github.com/anchore/syft) to generate a SBOM for the latest connector version and the one from the PR.
It allows us to spot the dependencies that have been updated at all levels and for all types of dependencies (system, python, java etc.).
Here are the dependencies that have been updated compared to `airbyte/source-monday:latest`.
Keep in mind that `:latest` does not always match the connector code on the main branch. 
It is the latest released connector image when the head commit of this branch was created.

| Type | Name | State | Previous Version | New Version |
|------|------|-------|-------------|------------------|
| python | CacheControl | updated | 0.13.1 | **0.14.2** |
| python | build | updated | 0.10.0 | **1.2.2.post1** |
| deb | libopenjp2-7 | updated | 2.5.0-2 | **2.5.0-2+deb12u1** |
| python | poetry | updated | 1.6.1 | **1.8.4** |
| python | poetry-core | updated | 1.7.0 | **1.9.1** |
| python | poetry-plugin-export | updated | 1.6.0 | **1.8.0** |
| binary | python | updated | 3.10.14 | **3.11.11** |
| python | wheel | updated | 0.44.0 | **0.45.1** |
| python | fastjsonschema | added | not present | **2.21.1** |
| python | exceptiongroup | removed | 1.2.2 | **not present** |
| python | tomli | removed | 2.2.1 | **not present** |

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=53828) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=53828) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=53828) it.</sub>

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
PATCH bump source-monday version to 2.3.1
Build source-monday docker image for platform(s) linux/amd64, linux/arm64
Get dependency updates
Create or update pull request on Airbyte repository
Add changelog entry

dbgold17 committed:
fixing python errors

dbgold17 commented on airbyte-integrations/connectors/source-monday/source_monday/extractor.py:
```diff
@@ -15,7 +15,7 @@
 from airbyte_cdk.sources.declarative.decoders.json_decoder import JsonDecoder
 from airbyte_cdk.sources.declarative.extractors.record_extractor import RecordExtractor
 from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
-from airbyte_cdk.sources.declarative.types import Config, Record
```
unused import

dbgold17 commented:
> /run-cat-tests
> 
> > Connector CAT tests started... [Check job output.](https://github.com/airbytehq/airbyte/actions/runs/15597771131)
> > If the connector supports live tests or regression tests, you may want to run those now also:
> > 
> > * [Run Live Tests](https://github.com/airbytehq/airbyte/actions/workflows/live_tests.yml)
> > * [Run Regression Tests](https://github.com/airbytehq/airbyte/actions/workflows/regression_tests.yml)
> > 
> > CAT Tests completed.
> > _**Important: Please check the job output to verify the results.**_

CATs seem to be passing, I'm good to merge this.

dbgold17 committed:
Merge branch 'master' into up-to-date/source-monday

ChristoGrab reviewed (approved):
LGTM! :shipit:
